### ISSUE TITLE
Not working Post-exploitation modules in the control panel

### ISSUE THREAD

**Issue by github-no-one-user** [malwaredllc/byob#547]
Hi Everyone,

I installed BYOB correctly (I suppose). If I use the shell I can do things on the infected PC.

However, none of the post-exploitation modules in the control panel works. Every time I get the error  `Bot 0: Error: client returned invalid response: "0"` 

What could be? Because the bot respond to shell commands, but BYOB does not let me use the post-exploitation modules...

Is it a problem in the side of the infected device or of the CNC device?

Thank you in advance

**Comment by rotzbouf** [comment]
Same results here. So I gave it a try in cli mode, which also failed, but since the modules seem to be the same (according to the last update time), I did not expected any other result.
Victim OS is one Ubuntu20.04 and another is Win10 (no other quick victims available right now)
Both are calling home when executing the compiled payload.

Could anyone prove working modules and provide the Target OS on which they worked?
